generateResources.py

```diff
@@ -980,6 +982,31 @@ def item(filename_parts, *layers, parent='item/generated'):
 # LEATHER / HIDES
 blockstate(('placed_hide',), 'tfc:hide_rack', {})
 
+# AGRICULTURE
+
+for crop in CROPS.keys():
+    stages = CROPS[crop]['stages']
+    texture = CROPS[crop]['texture']
+    mymodel = CROPS[crop]['model']
+
+    variants = {'stage': {}}
+    for stage in range(stages):
+        variants['stage'][stage] = {
+            'textures': {
+                texture: "tfc:blocks/crop/%s_%d" % (crop, stage)
+            }
+        }
```

Comment:
This can be reduced to
```python
{'stage': dict((str(i), { 'textures': { 'texture': 'tfc:blocks/crop%s_%d' % (crop, stage)}) for i in range(stages)}
```
and since it's only used once, just inline it in the method call

Reply:
While I do appreciate the nature of creating the dictionary in one call with the embedded for, we definitely disagree stylistically about the use of newlines. You love compressing code but I love separating code across lines.